src/extensions/default/SVGCodeHints/SVGAttributes.json

```diff
@@ -0,0 +1,372 @@
+{
+    "additive": {
+        "attribOptions": ["replace", "sum"]
+    },
+    "accumulate": {
+        "attribOptions": ["none", "sum"]
+    },
+    "alignment-baseline": {
+        "attribOptions": ["after-edge", "alphabetic", "auto", "baseline", "before-edge", "central", "hanging", "ideographic", "mathematical", "middle", "text-after-edge", "text-before-edge"]
+    },
+    "arabic-form": {
+        "attribOptions": ["initial", "isolated", "medial", "terminal"]
+    },
+    "attributeType": {
+        "attribOptions": ["auto", "CSS", "XML"]
+    },
+    "baseline-shift": {
+        "attribOptions": ["baseline", "inherit", "sub", "super"]
+    },
+    "calcMode": {
+        "attribOptions": ["discrete", "linear", "paced", "spline"]
+    },
+    "clip": {
+        "attribOptions": ["auto", "inherit"]
+    },
+    "clipPathUnits": {
+        "alias": "gradientUnits"
+    },
+    "clip-path": {
+        "attribOptions": ["inherit", "none"]
+    },
+    "clip-rule": {
+        "attribOptions": ["evenodd", "inherit", "nonzero"]
+    },
+    "color": {
```

Comment:
We should try to align this with [ColorUtils.js](https://github.com/adobe/brackets/blob/master/src/utils/ColorUtils.js).
Unfortunately, we only have a comprehensive regexp right now, including rgb, hex, ...

You should at least include `rebeccapurple`.